packages/commonwealth/client/scripts/models/Thread.ts

```diff
@@ -29,7 +29,21 @@ function processAssociatedContests(
   contestActions?: ContestActionT[] | null,
 ): AssociatedContest[] | [] {
   if (associatedContests) {
-    return associatedContests;
+    /**
+     * TODO: When we fix the content_id issue for 'added' contests, we should remove this deduplication logic
+     **/
+    const uniqueContestIds = new Set<string>();
+
+    const deduplicatedContests = associatedContests.filter((ac) => {
+      const key = `${ac.contest_id}:${ac.content_id}`;
```

Comment:
can we remove one instance of ac.contest_id?

Reply:
These are two separate variables `contest_id` and `content_id`. If that is not what you were referring to please let me know.

Reply:
oh haha, they looked so similar, mb